packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_types.dart

```diff
@@ -130,7 +132,7 @@ class EditableEnum extends EditableProperty with FiniteValuesProperty {
   }
 }
 
-class EditableProperty extends EditableArgument {
+class EditableProperty extends EditableArgument with SearchableDataMixin {
```

Comment:
Nit: is it worth mixing in SearchableDataMixin when this object is never used with any search functionality. You could inline the logic in matchesSearchToken into the filterData override instead.

Reply:
Ah I missed this, you're right! Removed the mixin